tests/unit/utils/sendTasksUpdates.test.ts

```diff
@@ -0,0 +1,33 @@
+import config from "../../../config/config";
+import JSONResponse from "../../../src/utils/JsonResponse";
+import { sendTaskUpdate } from "../../../src/utils/sendTaskUpdates";
+
+describe("sendTaskUpdate function", () => {
+  const mockEnv = { DISCORD_TOKEN: "mockToken" };
+  const completed = "Task completed successfully";
+  const planned = "Plan for the next phase";
+  const blockers = "No blockers";
+
+  test("should send the task update to discord tracking channel.", async () => {
```

Comment:
You can write tests for the undefined body keys. Like undefined content, undefined blockers, undefined completed, etc.

Reply:
Okay, I'll write that.

Reply:
Done sir, please check.